docs/mesos/README.md

```diff
@@ -0,0 +1,34 @@
+# Mesos with Calico Networking
+Calico provides networking for your Mesos cluster. These guides will get you started.
+
+## Prepare Master and Agent Nodes
+[The Host Preparation Guide](PrepareHosts.md) will walk you through preparing each host in your cluster to be compatible with Calico and Mesos.
+
+## Prepare External Services
+Zookeeper and etcd serve as the backend datastores for Mesos and Calico, respectively. [The External Services Preparation Guide](PrepareExternalServices.md) will walk you through setting up both services using Docker.
+
+## Mesos + Netmodules
+Calico works in conjunction with [Net-Modules][net-modules], a Mesos Networking Module. The following guides will install Mesos with the necessary Net-Modules libraries. 
+* Download and Install the Mesos + Net-Modules RPM (coming soon)
+* Create and Install the Mesos + Net-Modules RPM (coming soon)
+* Manually Compile and Install Mesos + Net-Modules (coming soon)
+
+## Calico
+The following guides will help you install the Calico services required for each agent in your Mesos cluster.
+
+### Calico With Docker (Recommended)
+Calico is primarily distributed as a Docker container. The following guides will help you set that up.
+* Download and Install the Calico RPM (coming soon)
```

Comment:
Should we wait to mark it recommended until we have a guide and release with the calico-mesos rpms?